src/slots.rs

```diff
@@ -122,6 +122,8 @@ impl<'a> SkippedSlotsMonitor<'a> {
             };
             let skipped_count = get_count(SlotStatus::Skipped);
             let validated_count = get_count(SlotStatus::Validated);
+            // total_count > 0 since either skipped_count > 0 or validated_count > 0. Hence the
+            // result of division by total_count is always defined.
```

Comment:
Perhaps an `assert!` would be useful here. [This may allow the compiler, in some cases, to perform special optimisations!](https://gist.github.com/jFransham/369a86eff00e5f280ed25121454acec1#assert-conditions-beforehand)